shared_libraries/orca_shared/recovery/test/unit_tests/test_shared_recovery.py

```diff
@@ -55,18 +55,18 @@ def tearDown(self):
         {"AWS_REGION": "us-west-2"},
         clear=True,
     )
-    def test_post_entry_to_queue_no_errors(self):
+    def test_post_entry_to_fifo_queue_no_errors(self):
         """
         *Happy Path*
-        Test that sending a message to SQS queue using post_entry_to_queue()
+        Test that sending a message to SQS queue using post_entry_to_fifo_queue()
         function returns the same expected message.
         """
         new_data = {"name": "test"}
         for request_method in self.request_methods:
             # Run subtests
             with self.subTest(request_method=request_method):
                 # Send values to the function
-                shared_recovery.post_entry_to_queue(
+                shared_recovery.post_entry_to_fifo_queue(
                     new_data, request_method, self.db_queue_url
```

Comment:
We might want a card on the backlog to add to these tests. Hopefully there's a way to check that request_method is used.

Reply:
i think request_method is captured in MessageAttributes and then checked

                queue_contents = self.queue.receive_messages(
                    MessageAttributeNames=["All"]
                )
                queue_output_body = json.loads(queue_contents[0].body)

                # Testing SQS body
                self.assertEqual(queue_output_body, new_data)

Reply:
Nope. Note how new_data is just `{"name": "test"}`
We'll want to check the error cases eventually as well.

Reply:
sounds good. created [ORCA-366](https://bugs.earthdata.nasa.gov/browse/ORCA-366) for additional tests.